eitprocessing/eit_data/timpel.py

```diff
@@ -88,43 +88,76 @@ def _from_path(  # pylint: disable=too-many-arguments,too-many-locals
 
         pixel_impedance = data[:, :1024]
         pixel_impedance = np.reshape(pixel_impedance, newshape=(-1, 32, 32), order="C")
-        pixel_impedance = np.where(pixel_impedance == -1000, np.nan, pixel_impedance)
+
+        pixel_impedance = np.where(pixel_impedance == _NAN_VALUE, np.nan, pixel_impedance)
 
         # extract waveform data
         # TODO: properly export waveform data
-        waveform_data = {  # noqa
-            "airway_pressure": data[:, 1024],
-            "flow": data[:, 1025],
-            "volume": data[:, 1026],
-        }
+
+        continuous_data_collection = DataCollection(ContinuousData)
+        continuous_data_collection.add(
+            ContinuousData(
+                "airway_pressure_(timpel)",
+                "Airway pressure",
+                "cmH2O",
+                "pressure",
+                "Airway pressure measured by Timpel device",
+                loaded=True,
+                values=data[:, 1024],
+            ),
+        )
+
+        continuous_data_collection.add(
+            ContinuousData(
+                "flow_(timpel)",
+                "Flow",
+                "L/s",
+                "flow",
+                "FLow measures by Timpel device",
+                loaded=True,
+                values=data[:, 1025],
+            ),
+        )
+
+        continuous_data_collection.add(
+            ContinuousData(
+                "volume_(timpel)",
+                "Volume",
+                "L",
+                "volume",
+                "Volume measured by Timpel device",
+                loaded=True,
+                values=data[:, 1026],
+            ),
+        )
 
         # extract breath start, breath end and QRS marks
         phases = []
         for index in np.flatnonzero(data[:, 1027] == 1):
-            phases.append(MinValue(index, time[int(index)]))
+            phases.append(MinValue(index, time[int(index)]))  # noqa: PERF401
 
         for index in np.flatnonzero(data[:, 1028] == 1):
-            phases.append(MaxValue(index, time[int(index)]))
+            phases.append(MaxValue(index, time[int(index)]))  # noqa: PERF401
 
         for index in np.flatnonzero(data[:, 1029] == 1):
-            phases.append(QRSMark(index, time[int(index)]))
+            phases.append(QRSMark(index, time[int(index)]))  # noqa: PERF401
```

Comment:
shouldn't these be loaded as SparseData? Or are we not handling those as such yet?